ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.css

```diff
@@ -58,19 +58,11 @@ div#topology-resize-panel table td {
     fill: #6853ae;
 }
 
-.derived-namespace .pf-topology__node__label__background {
+.related-namespace.pf-m-selected text, .related-namespace.pf-m-selected .pf-topology__node__action-icon svg {
     fill: #ffffff;
 }
 
-.derived-namespace > g > text, .derived-namespace .pf-topology__node__action-icon svg {
-    fill: #000000;
-}
-
-.derived-namespace.pf-m-selected text, .derived-namespace.pf-m-selected .pf-topology__node__action-icon svg {
-    fill: #ffffff;
-}
-
-.derived-namespace path.pf-topology__group__background {
+.related-namespace path.pf-topology__group__background {
```

Comment:
I'm not a fan of always changing internal names, every time there's an editorial change to a displayed name. It's fine  since it already done here, but it can make for extremely noise DIFFs, the more places an internal identifier is used.

Reply:
True but then we'd have multiple different names for the same thing, which could be confusing (and definitely was) 😓